### DISCUSSION TITLE
Medley Interlisp Users/Interest vs GitHub Discussions

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1036] [General]
What user topics not directly related to Medley's development are better discussed in the [Medley Interlisp Users/Interest](https://groups.google.com/g/interlisp) mailing list and which in the project's [GitHub discussion board](https://github.com/Interlisp/medley/discussions)?

I'm looking for a place to discuss topics and issues regarding the ordinary use of Medley by the users, such as seeking information and advice, asking for help, learning and programming Medley, clarifying the documentation, sharing tips and tricks, operating the system and performing tasks, troubleshooting usage issues that are not Medley bugs, and so on.

**Comment by masinter** [comment]
@pamoroso I'd suggest looking at the Issues labeled "documentation" AND also issues in the Interlisp/Interlisp.github.io repo. 
- Getting a review of the issues for common themes, overlaps enough that separating them was hard
- I think the https://github.com/Interlisp/medley/discussions switch wasn't helpful -- it just made discussions of topics harder to find.

If you'd like to zoom to talk about this, either on the Monday agenda or we can schedule a separate meeting depending on your timezone.

**Comment by pamoroso** [comment]
Thanks, the GitHub issues are a great learning and information resource. For example, I was wondering how to launch the documentation browser other than from the background menu, then stumbled upon an [issue](https://github.com/Interlisp/medley/issues/1030#issuecomment-1368318478) where you mentioned `HELPSYS`.

So is creating issues tagged `documentation` a good way of discussing general user issues and asking questions? What's the  Medley Interlisp Users/Interest mailing list best for instead?

**Reply by nbriggs** [reply]
The choice of Github discussions or the user/interest mailing list for finding answers is more a matter of *your* work style than anything we dictate.  I would reserve actually opening an issue (documentation or otherwise) for situations where it requires something to be fixed -- that might be an error (in the code or documentation), something that ought to be documented but isn't, or if there is no way to find a path into the existing documentation from any known or searchable starting point.

In keeping with original Interlisp style... there's never (seldom?) just one way to do something, and sometimes that causes more work because not only does one have to explain multiple individual solutions but one has to also explain the meta-point of how to choose between them.

**Reply by pamoroso** [reply]
Thanks, these are useful guidelines.

**Comment by rainthree** [comment]
you are invited here https://discord.gg/eXzdWnAr , a discord chat server which is created exactly for this "I'm looking for a place to discuss topics and issues regarding the ordinary use of Medley by the users, such as seeking information and advice, asking for help, learning and programming Medley, clarifying the documentation, sharing tips and tricks, operating the system and performing tasks, troubleshooting usage issues that are not Medley bugs, and so on."

**Reply by faeredia** [reply]
"Invite Invalid" when I click on that, do you mind refreshing?

**Comment by masinter** [comment]
Does Discord have an option of recording meetings with video? We should look into whether it would be a better choice than Zoom

**Comment by pamoroso** [comment]
I'd prefer not to touch Discord at all as its user interface feel busy and overwhelming.

**Comment by masinter** [comment]
GitHub and the interlisp.org web site are our primary means of coordiniation and communication.
The mailing lists/Google Groups Interlisp and LispCore @googlegroups.com, info@interlisp.org, tweet to @interlisp8 , IRC (Libera Chat), Discord and so on are secondary because some people prefer them while others abhor. What I think we need is to use the "What people are saying?" section of the web site to cross-post summaries of other postings.Perhaps we need a "social media coordinator"?  @zbraiterman ? What do you think?

**Comment by zbraiterman** [comment]
This sounds good to me, @masinter.  I'm flattered that you thought of me.  I'm happy to help.
